sonar-project.properties

```diff
@@ -0,0 +1,12 @@
+sonar.projectKey=matrix-react-sdk
+sonar.organization=matrix-org
+
+# This is the name and version displayed in the SonarCloud UI.
+#sonar.projectName=matrix-react-sdk
+#sonar.projectVersion=1.0
+
+# Path is relative to the sonar-project.properties file. Replace "\" by "/" on Windows.
+#sonar.sources=.
+
+# Encoding of the source code. Default is default system encoding
+#sonar.sourceEncoding=UTF-8
```

Comment:
I'm excited about trialing sonar!
It's a maze of properties, and it's worth the time to try to fine-tune them to leverage the tool as much as we can https://docs.sonarcloud.io/advanced-setup/analysis-parameters/ 

The ones that I found really useful from the beginning were:
- Setting up the tests folder `sonar.tests`
- Setting up the source folder `sonar.sources`
- Exclude some folders with `sonar.exclusions`